src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -53,38 +64,13 @@ class HaConfigDashboard extends LitElement {
             ${this.hass.localize("ui.panel.config.introduction")}
           </div>
 
-          ${this.cloudStatus && isComponentLoaded(this.hass, "cloud")
```

Comment:
Please restore this.

Reply:
Cloud is still there?
https://github.com/home-assistant/home-assistant-polymer/pull/4519/files#diff-c0ff620c0eeedb080ffe9e78fce06579R71
`ha-config-navigation` has more or less the identical logic as the stuff I removed here, looked weird to have the same logic in both places, but sure I can revert this.

Reply:
You would render an empty card now, place this check around the `ha-card`